peekingduck/nodes/output/media_writer.py

```diff
@@ -51,17 +51,23 @@ class Node(AbstractNode):
     Configs:
         output_dir (:obj:`str`): **default = "PeekingDuck/data/output"**. |br|
             Output directory for files to be written locally.
+        output_filename (:obj:`str`): **default = None**. |br|
```

Comment:
The default is a `null`, not `None`.
c.f. `dabble.statistics` node's docstrings.